.github/workflows/build_conda_linux.yml

```diff
@@ -114,6 +114,18 @@ jobs:
           else
             ${CONDA_RUN} bash "${POST_SCRIPT}"
           fi
+      - name: Smoke Test
+        working-directory: ${{ inputs.repository }}
+        run: |
+          #./test/smoke_test/run_smoke_test.sh
+          set -ex
+          #conda create -yp test-luck python=3.10 numpy
+          #conda run -p test-luck conda install pytorch -c pytorch-nightly
+          ${CONDA_RUN} conda install -v -y -c file://$PWD/dist torchaudio
```

Comment:
See tips here: https://github.com/conda/conda/issues/7024#issuecomment-447517963 in case this command isn't working